backend/src/main/java/gov/cdc/ReportVision/models/Page.java

```diff
@@ -0,0 +1,34 @@
+package gov.cdc.ReportVision.models;
+
+import jakarta.persistence.*;
+import lombok.Getter;
+
+import java.util.List;
+import java.util.UUID;
+
+@Getter
+@Entity
+public class Page extends BaseEntity {
+    @Id
+    @GeneratedValue(generator = "UUID4")
+    @Column(updatable = false, nullable = false)
+    private UUID id;
+
+
+    @JoinTable(
+        name = "page_labels",
+        joinColumns = @JoinColumn(name = "page_id"),
+        inverseJoinColumns = @JoinColumn(name = "label_id"))
+    @OneToMany
+    private List<Label> labels;
+
+    //base64 image
+    private String sourceImage;
+    //base 64 image
+    private String templateImage;
+    //json array
+    private String shapes;
```

Comment:
@Column(columnDefinition = "jsonb") for explicit native JSON Storage

Reply:
Zedd and I discussed this, stuck with String until such a time as we have a reason to change it. His primary concern was that `jsonb` columns will force a check that it's valid JSON and we may not need this. I have a hunch that it's something we'll end up wanting but that can be a migration when we need it (likely in #266) rather than have to sort it all out ahead of time.